modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc

```diff
@@ -1,21 +1,110 @@
 :page-aliases: cloud:dedicated-byoc.adoc, deploy:deployment-option/cloud/dedicated-byoc.adoc
 :page-cloud: true
 
-Redpanda supports <<Dedicated Cloud>>, with
-clusters running in Redpanda Cloud, and also <<BYOC, Bring Your Own Cloud (BYOC)>>,
-with clusters running in your private cloud. Redpanda
-Cloud continuously monitors and maintains your Redpanda clusters along with the
-underlying Redpanda Cloud infrastructure to meet strict performance, availability,
-reliability, and security requirements. These clusters are also deployed with an
-integrated and dedicated Redpanda Console and, optionally, with a fully managed
-Kafka Connect and connectors cluster.
+Redpanda Cloud is a complete streaming data platform delivered as a fully-managed service with automated upgrades and patching, data and partition balancing, and 24x7 support. It continuously monitors and maintains your clusters along with the underlying infrastructure to meet strict performance, availability, reliability, and security requirements. All Redpanda Cloud clusters are deployed with an integrated glossterm:Redpanda Console[].
+
+== Cluster types
+
+Redpanda offers three types of fully-managed cloud clusters:
+
+* <<Serverless>>: Clusters hosted in Redpanda Cloud. This is the fastest and easiest way to start data streaming in the cloud. 
+* <<Dedicated Cloud>>: Single-tenant clusters hosted in Redpanda Cloud. This provides more control over your deployment. 
+* <<Bring Your Own Cloud (BYOC)>>: Clusters hosted in your private cloud. This provides more control over your deployment and offers full data sovereignty. 
+
+NOTE: With standard BYOC clusters, Redpanda manages security policies and resources for your VPC, including service accounts, subnetworks, IAM roles, firewall rules, and storage buckets. For the most security, you can manage these resources yourself with a xref:./vpc-byo-gcp.adoc[customer-managed VPC].
+
+=== Serverless
+
+With Serverless clusters, you host your data in Redpanda's VPC, and Redpanda handles automatic scaling, provisioning, operations, and maintenance. This is a production-ready deployment option with a cluster available instantly. This is no base cost, and with pay-as-you-go billing after the free trial, you only pay for what you consume. 
+
+Available on AWS.
+
+NOTE: Serverless is currently in a limited availability (LA) release with xref:deploy:deployment-option/cloud/serverless.adoc#limits[usage limits] and no public SLA. During LA, existing clusters can scale to the usage limits, but new clusters may need to wait for availability.
+
+==== Sign up for Serverless
+
+To start using Serverless, sign up for a free trial. New trials receive $100 (USD) in free credits to spend in the first 14 days. This should be enough to run Redpanda with reasonable throughput. No credit card is required for a trial. To continue using Serverless, add a credit card and pay as you go. You can view invoices and edit payment methods on the Redpanda Cloud *Billing* page. 
```

Comment:
> You can view invoices

I don't think they'll be able to view invoices for LA. They will be sent invoices by Stripe initially.